### PR TITLE
General improvements for iter8.tools text and Iter8 repo issue/PR templates

### LINKED ISSUE #756: [Documentation Request] Improvements to the text of the documentation
**Description:**

General improvements for https://iter8.tools and at the Iter8 repo.

1. Concepts markdowns needs to be better aligned with the landing page text.
2. Task descriptions need to be improved in terms of clarity and also spelling/grammar. I found a number of errors that relate to YAML formatting in task descriptions.
3. Issue and PR templates for the Iter8 repo need fixing. They are outdated and beginning to show the limits.

### PR DESCRIPTION
**Pull request by sriumcp**
Fixes #756

### CODE REVIEW

File: mkdocs/docs/reference/tasks/metrics.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+template: main.html
+---
+
+# Metrics Tasks
+
+## `metrics/collect`
+
+### Overview
+
+The `metrics/collect` takes enables collection of [builtin metrics](). It generates a stream of HTTP requests to one or more app/ML model versions, and collects latency/error metrics.
```
Comment (kalantar) [inline mkdocs/docs/reference/tasks/metrics.md]: missing link

File: mkdocs/docs/reference/tasks/metrics.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	Versions []Version `json:"versions" yaml:"versions"`
+	// URL of the JSON file to send during the query; optional
+	PayloadURL *string `json:"payloadURL,omitempty" yaml:"payloadURL,omitempty"`
+} -->
+
+| Field name | Field type | Description | Required |
+| ----- | ---- | ----------- | -------- |
+| time | string | Duration of the `metrics/collect` task run. Specified in the [Go duration string format](https://golang.org/pkg/time/#ParseDuration). Default value is `5s`. | No |
+| payloadURL | string | URL of JSON-encoded data. If this field is specified, the metrics collector will send HTTP POST requests to versions, and the POST requests will contain this JSON data as payload. | No |
+| versions | [][Version](#version) | A non-empty list of versions. | Yes |
```
Comment (kalantar) [inline mkdocs/docs/reference/tasks/metrics.md]: you added a parameter `loadOnly`

**Review by kalantar — CHANGES_REQUESTED** [review]
minor changes. Also I see  mkdocs/docs/metrics/builtin.md is an empty file. Is this intended?

**Comment by sriumcp** [comment]
@kalantar Sorry, this one is not yet ready for review :) I should have made this into a draft... doing so now. Will be able to submit this for review shortly (and will address all your comments hopefully by then).

**Comment by huang195** [comment]
@sriumcp Is there no way to break this PR into smaller chunks? I'm looking at all the changes, and it's kind of difficult for me to understand how all the changes are related and be able to review them. If there's no clear way to break them into smaller pieces, then that's fine. It will just take longer to review the changes.

**Comment by sriumcp** [comment]
@kalantar @huang195 Let me break it up into multiple chunks as suggested. I will convert  this into draft meanwhile to facilitate this.

**Comment by sriumcp** [comment]
@kalantar @huang195  This one is ready for review.

**Review by kalantar — DISMISSED** [review]
Approved but recommend changing name of PR. It no longer hand to do with builtin metrics.

**Comment by sriumcp** [comment]
@kalantar PR renamed.

**Pull request merged**